x-pack/platform/packages/shared/kbn-ai-assistant/src/utils/create_mock_chat_service.ts

```diff
@@ -25,13 +24,7 @@ export const createMockChatService = (): MockedChatService => {
     hasFunction: jest.fn().mockReturnValue(false),
     hasRenderFunction: jest.fn().mockReturnValue(true),
     renderFunction: jest.fn(),
-    getSystemMessage: jest.fn().mockReturnValue({
-      '@timestamp': new Date().toISOString(),
-      message: {
-        role: MessageRole.System,
-        content: '',
-      },
-    }),
+    getSystemMessage: jest.fn().mockReturnValue(''),
```

Comment:
Should this not contain a system message?

```suggestion
    getSystemMessage: jest.fn().mockReturnValue('system message'),
```

Reply:
done